packages/arb-validator-core/ethbridge/arbRollupWatcher.go

```diff
@@ -22,6 +22,8 @@ import (
 	"math/big"
 	"strings"
 
+	"github.com/offchainlabs/arbitrum/packages/arb-validator-core/ethbridge/globalinbox"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 88 characters, out of limit 80](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)